tutornotes/patches/local-docker-compose-jobs-services

```diff
@@ -5,4 +5,4 @@ notes-job:
   volumes:
     - ../plugins/notes/apps/settings/tutor.py:/openedx/edx-notes-api/notesserver/settings/tutor.py:ro
   depends_on:
-    - mysql
\ No newline at end of file
+    {% if ACTIVATE_MYSQL %}- mysql{% endif %}
```

Comment:
This will fail if `ACTIVATE_MYSQL` is false, as the `depends_on` key will then be `None` instead of an empty array. You should write instead:

    depends_on: {{ [("mysql", ACTIVATE_MYSQL)]|list_if }}

And same below.

Reply:
Thank, Will do that now.